src/main/resources/applicationConfig.xml

```diff
@@ -47,8 +47,6 @@
     <defaultProjectId>global-default</defaultProjectId>
     <defaultStorageName>global</defaultStorageName>
     <maxLookBackInDays>365</maxLookBackInDays>
-    <throttleLimit>30</throttleLimit>
```

Comment:
can we update/remove this setting from the associated readme guide - https://github.com/GoogleCloudPlatform/storage-sdrs/blob/master/readme/README-executor.md

Reply:
done